packages/components/src/internal/app/utils.ts

```diff
@@ -139,7 +143,11 @@ export function isSampleAliquotEnabled(): boolean {
 
 export function hasPremiumModule(): boolean {
     const { moduleContext } = getServerContext();
-    return useMemo(() => moduleContext?.samplemanagement?.hasPremiumModule ?? false, [moduleContext]);
+    return  moduleContext?.samplemanagement?.hasPremiumModule ?? false;
```

Comment:
@labkey-susanh  I believe Nick had suggested that I add this hook in this case. Was it causing you issues in some scenario?

Reply:
Yes, it does not work unless called from a functional component, and I used it in the new utility method for `isCommunityDistribution`, which is sometimes not called from a functional component.  In general, it seems sort of a bad idea for utility methods to use hooks, but maybe I'm missing something.